inc/core/admin.php

```diff
@@ -86,92 +84,120 @@ public function incompatibility_notice() {
 		if ( $current_screen->id === 'appearance_page_neve-welcome' ) {
 			return;
 		}
-
+		$theme_name  = apply_filters( 'ti_wl_theme_name', $this->theme_args->__get( 'Name' ) );
```

Comment:
Fixes the white-label module filter (it was previously called to early in the procedural execution of the application).